src/Microsoft.Framework.ConfigurationModel/ConfigurationExtensions.cs

```diff
@@ -47,5 +48,10 @@ public static IConfigurationSourceRoot AddEnvironmentVariables(this IConfigurati
             configuration.Add(new EnvironmentVariablesConfigurationSource(prefix));
             return configuration;
         }
+
+        public static string ResolvePath(this IConfigurationSourceRoot configuration, string path)
+        {
+            return Path.Combine(configuration.BasePath ?? string.Empty, path);
```

Comment:
`BasePath` will never be `null` if you follow what @davidfowl [said](https://github.com/aspnet/Configuration/pull/160/files#r26820907)